config.m4

```diff
@@ -421,6 +421,7 @@ if test "$PHP_MONGODB" != "no"; then
       dnl Generated with: find src/libmongocrypt/kms-message/src -maxdepth 1 -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
       PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES="hexlify.c kms_b64.c kms_caller_identity_request.c kms_crypto_apple.c kms_crypto_libcrypto.c kms_crypto_none.c kms_crypto_windows.c kms_decrypt_request.c kms_encrypt_request.c kms_kv_list.c kms_message.c kms_request.c kms_request_opt.c kms_request_str.c kms_response.c kms_response_parser.c sort.c"
 
+      PHP_MONGODB_LIBMONGOCRYPT_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS $PHP_MONGODB_LIBMONGOCRYPT_CFLAGS"
```

Comment:
Since [`libmongocrypt/CheckSSL.m4`](https://github.com/mongodb/mongo-php-driver/blob/master/scripts/autotools/libmongocrypt/CheckSSL.m4) relies on the `$PHP_MONGODB_SSL` var set in [`libmongoc/CheckSSL.m4`](https://github.com/mongodb/mongo-php-driver/blob/master/scripts/autotools/libmongoc/CheckSSL.m4), we're always assured that libmongocrypt and libmongoc are using the _same_ TLS library. This change was just to defer reading of `$PHP_MONGODB_BUNDLED_CFLAGS` until _after_ any m4 includes might modify it.

Reply:
Correct as well. libmongocrypt/CheckSSL.m4 never did its own SSL checking, it merely assigned compiler flags to ensure that libmongocrypt is built with the same SSL library we've selected for libmongoc. I'm surprised it took this long to figure out this issue, since the default includes seem to be sufficient in most cases to locate SSL headers. Brief stored those in `/opt/bref/includes`, which wasn't picked up for libmongocrypt compilation and thus caused failures even though OpenSSL was present and found.